functional-tests/mine_once_test.go

```diff
@@ -0,0 +1,61 @@
+package functional
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-filecoin/build/project"
+	"github.com/filecoin-project/go-filecoin/internal/app/go-filecoin/porcelain"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/block"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers/iptbtester"
+	tf "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers/testflags"
+)
+
+func TestBootstrapMineOnce(t *testing.T) {
+	tf.FunctionalTest(t)
+
+	ctx := context.Background()
+	root := project.Root()
+
+	tns, err := iptbtester.NewTestNodes(t, 1, nil)
```

Comment:
What's the reason to use this instead of fast?  Do other functional tests use this?  Is fast setup no good for our purposes here?  Fast is too advanced a tool for what we want?  Only asking because I've never seen this integration testing framework used directly like this and we already have a proliferation of integration testing methods (in process tools, test daemon, fast)

Reply:
TBH, I started with the first functional test in the functional tests directory and it didn't happen to be a fast test. I'll look into what it would take to convert it.

Reply:
I looked into it. It would require quite a bit of fast modifications to get its initialization to support presealed sectors. I'm going to merge this and revisit for the next functional test.